vortex-metrics/src/lib.rs

```diff
@@ -144,3 +168,120 @@ impl<'a> Iterator for VortexMetricsIter<'a> {
         self.iter.size_hint()
     }
 }
+
+impl DefaultTags {
+    fn merge(&self, other: &Self) -> Self {
```

Comment:
I think this is a bit confusing, because it'll take values from the `self` over values from `other`, not merging them (if they share keys).

Reply:
nvm, took a deeper look and I think this seems right, I'm just not sure which value it'll actually take but probably `other` (which is what we want).